WalletWasabi/WabiSabi/Models/CoordinationFeeRate.cs

```diff
@@ -0,0 +1,20 @@
+using NBitcoin;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.WabiSabi.Models
+{
+	public struct CoordinationFeeRate
+	{
+		public static readonly CoordinationFeeRate Zero = new(0);
+
+		public CoordinationFeeRate(decimal rate)
```

Comment:
That's a sanity check just in case something goes wrong so neither the coordinator or the clients will accept a coordination fee rate greater than 0.01.

About percentages, it is common for people to use percentages for everything even for probabilities and rates because it lowers the language to a level that everybody can grasp but that is incorrect because in order to be useful we need to convert those percentages to rates in order to be able to use them anyway. Here we deal with rates all time when we pay network fees and the same should be for coordination fees.